models/cart-discount/src/cart-discount/cart-discount-draft/presets/sample-data-goodstore/furniture-bogo.ts

```diff
@@ -0,0 +1,54 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import {
+  ProductTypeDraft,
+  TProductTypeDraft,
+} from '@commercetools-test-data/product-type';
+import {
+  CartDiscountMultiBuyLineItemsTargetDraft,
+  CartDiscountValueRelativeDraft,
+  selectionMode,
+} from '../../../../index';
+import { stackingMode } from '../../../constants';
+import type { TCartDiscountDraftBuilder } from '../../../types';
+import * as CartDiscountDraft from '../../index';
+
+const productTypeDraft = ProductTypeDraft.presets.sampleDataGoodStore
```

Comment:
Does this need to be `ProductTypeDraft.draftPresets.sampleDataGoodStore`?

Reply:
I don't see results for `.draftPresets`, but you're right to indicate that these don't exist as of now. I've stubbed them with a TODO in the meantime